About.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about.css">
+    <title>Document</title>
+</head>
+<body>
+    <main header>
+      <div class="nav">
+      <img 
+      src="Asset/feat img/Icon - Menu.png"
+      alt="hamburger-icon-menu"
+      class="nav-btn humbergenIcon"
+      id="hamburger-btnicon"
+    />
+    <img
+      src="Asset/feat img/IconCancel.png"
+      alt="Close-icon-menu"
+      class="close-icon hidden"
+      id="close-btnicon"
+    />
+  </div>
+        <div class="site-content">
+          <div class="wanted">
+              <div>
+              <img class="lol" src="" alt="counter" />
+              </div>
+              <ul class="wina">
+              <li>Logout</li>
+              <li>Mypage</li>
```

Comment:
Kindly adjust the about page to be as close to the design as possible.

Your design:
![image](https://github.com/maserekafelex/Capston-project/assets/20554583/545215dd-4801-4e4f-8078-b8cfdb6c1e86) ![image](https://github.com/maserekafelex/Capston-project/assets/20554583/78c6a7db-f80d-4da3-9b1e-ec13bae7b3b6) ![image](https://github.com/maserekafelex/Capston-project/assets/20554583/bb570a7d-b1a7-45cd-8c6a-cefef1bcdd77)

Design Guide:
![image](https://github.com/maserekafelex/Capston-project/assets/20554583/4406501a-4bc5-4181-9db3-2ca786a05036)
![image](https://github.com/maserekafelex/Capston-project/assets/20554583/ab8c69ef-6b06-450d-9e8e-e7673409f0a7)
![image](https://github.com/maserekafelex/Capston-project/assets/20554583/e9e9f236-71da-473c-bbed-f2feb060b120)